gpg-interface.c

```diff
@@ -88,6 +88,9 @@ int check_signature(const char *payload, size_t plen, const char *signature,
 
 void print_signature_buffer(const struct signature_check *sigc, unsigned flags)
 {
+	if (flags & GPG_VERIFY_QUIET)
```

Comment:
What do you mean by *the tag code*? Your initial patch? You are definitely right, but some code has to be changed in order add functionality.